src/it/java/teammates/it/sqllogic/api/EmailGeneratorTestIT.java

```diff
@@ -106,12 +113,14 @@ public void testGenerateSessionLinksRecoveryEmail() throws Exception {
         Student student1InCourse4 = dataBundle.students.get("student1InCourse4");
 
         email = emailGenerator.generateSessionLinksRecoveryEmailForStudent(
-                student1InCourse4.getEmail());
+                student1InCourse4.getEmail(), nonExistentStudent, emptyFragmentList);
 
         subject = EmailType.SESSION_LINKS_RECOVERY.getSubject();
 
         verifyEmail(email, student1InCourse4.getEmail(), subject,
                 "/sessionLinksRecoveryOpenedOrClosedAndpublishedSessions.html");
+
+        ______TS("Migration test: students from both postgres and datastore are fetched");
```

Comment:
Are we doing this here? If not I think can remove first

Reply:
Yup my bad! Didn't want to mock datastore values here, so I'll add it as a unit test in another PR